README.md

```diff
@@ -836,6 +778,70 @@ utils.distributed_cmd_run(train)
 </p>
 </details>
 
+<details>
+<summary>CV - MNIST with Metric Learning</summary>
+<p>
+
+```python
+from torch.optim import Adam
+from torch.utils.data import DataLoader
+
+import catalyst.contrib.data.transforms as t
+from catalyst.contrib.datasets.mnist import MnistMLDataset, MnistQGDataset
+from catalyst.contrib.dl.callbacks import PeriodicLoaderCallback
+from catalyst.contrib.models.simple_conv import SimpleConv
+from catalyst.contrib.nn.criterion.triplet import TripletMarginLossWithSampling
+from catalyst.core.callbacks import ControlFlowCallback
+from catalyst.data.sampler import BalanceBatchSampler
+from catalyst.data.sampler_inbatch import AllTripletsSampler
+from catalyst.dl import CriterionCallback
+from catalyst.dl.callbacks.metrics.cmc import CMCScoreCallback
+from catalyst.dl.runner import SupervisedRunner
```

Comment:
let's just use `from catalyst import dl` and use Callbacks, Runners, like `dl.{something}`

Reply:
done